test/kata7.getEmployerRole.test.js

```diff
@@ -1,5 +1,10 @@
 const { getEmployerRole } = require("../src");
 
 describe("getEmployerRole", () => {
-  test("returns the employee's role in the company", () => {});
+  it("returns the employee's role in the company", () => {
+  const employees = [{name: 'Satti',role: 'Developer'}, {name: 'Jenny',role: 'Sales Associate'}, {name: 'Javid',role: 'Human Recommended Reading Assistant'}];
```

Comment:
Formatting for better readability
````
    const employees = [
      { name: "Satti", role: "Developer" },
      { name: "Jenny", role: "Sales Associate" },
      { name: "Javid", role: "Human Recommended Reading Assistant" },
    ];
`````
And the single quotation marks should be "" .